packages/demand-deeplink/src/parse.ts

```diff
@@ -177,9 +178,16 @@ function parseSearchString(query: string) {
   const data = new URLSearchParams(query)
 
   data.forEach((v, k) => {
-    const value = v && v.trim()
+    let value = v && v.trim()
     const key = k && k.trim().toLowerCase()
 
+    if (brokenTimeFormatRegexp.test(value)) {
```

Comment:
Looks good, let's only add a comment with example and explanation why we are doing it